### DISCUSSION TITLE
elastalert alert to zabbix

### DISCUSSION THREAD

**Discussion by elktty** [jertel/elastalert2#438] [Q&A] [answered]
Hello!
Has anyone come across such a question?

I send alert to zabbix and in zabbix last data i get value "1"
![image](https://user-images.githubusercontent.com/89647230/131508419-03f36a6b-fbd9-487f-b4db-81d01ef0713a.png)

Here is an example of data that comes to my mail
![image](https://user-images.githubusercontent.com/89647230/131517654-0cbf3904-ddfb-4485-a819-11a86121d765.png)

Can I receive all the data sent by elastalert?

**Comment by nsano-rururu** [comment, marked as answer]
We are sending alerts using py-zabbix in python library. The current movement is as specified, so there is nothing we can do about it. It's not a bug.

Source code for current zabbix alert notifications
https://github.com/jertel/elastalert2/blob/master/elastalert/alerters/zabbix.py

**Reply by elktty** [reply]
Understood thanks
